kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -546,11 +585,17 @@ define(['common/html', 'common/format', 'common/ui', 'common/errorDisplay'], (
                         )
                     );
                 })
-                .join(', ')
-        );
+                .join(', ');
+
+        const jobSpan = document.createElement('span');
+        jobSpan.innerHTML = summaryHtml;
+        jobSpan.title = jobSpan.textContent;
+        return jobSpan.outerHTML;
```

Comment:
add a title element with the whole job status string in case it gets cut off behind the cell tabs

<img width="782" alt="title-element" src="https://user-images.githubusercontent.com/556057/115800369-506e7880-a38f-11eb-9196-8cdb266bedd7.png">